lib/spreewald_support/tolerance_for_selenium_sync_issues.rb

```diff
@@ -38,22 +38,26 @@ class Patiently
     WAIT_PERIOD = 0.05
 
     def patiently(seconds, &block)
-      started = Time.now
+      started = monotonic_time
       tries = 0
       begin
         tries += 1
         block.call
       rescue Exception => e
         raise e unless retryable_error?(e)
-        raise e if (Time.now - started > seconds && tries >= 2)
+        raise e if (monotonic_time - started > seconds && tries >= 2)
         sleep(WAIT_PERIOD)
-        raise Capybara::FrozenInTime, "time appears to be frozen, Capybara does not work with libraries which freeze time, consider using time travelling instead" if Time.now == started
+        raise Capybara::FrozenInTime, "time appears to be frozen, Capybara does not work with libraries which freeze time, consider using time travelling instead" if monotonic_time == started
         retry
       end
     end
 
     private
 
+    def monotonic_time
```

Comment:
Please elaborate in a comment why it's necessary to use monotonic time instead of Time.now